CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ for further information about branching and tagging conventions.
 ## [Unreleased]
 #### Fixed
 - Fixed operation empty and loading list displaying of a user
+- Add filter for fill limits in reviewLimitsUpdateRequest when 
+parsing limits values
```

Comment:
cannot get the meaning out of this. can you pls rephrase it? what actually was fixed?